queue_job/models/queue_job.py

```diff
@@ -231,6 +231,12 @@ def _compute_graph_jobs_count(self):
 
     @api.model_create_multi
     def create(self, vals_list):
+        if not vals_list:
+            return super(
+            QueueJob,
+            self.with_context(mail_create_nolog=True, mail_create_nosubscribe=True),
+        ).create(vals_list)
+
         if self.env.context.get("_job_edit_sentinel") is not self.EDIT_SENTINEL:
```

Comment:
so the new block above is the same as adding a new predicate `if vals_list and` in this line no?